tools/reports/backup/files.js

```diff
@@ -19,9 +19,9 @@ module.exports = {
           // Possibly extract objects.
           if (extract) {
             extractFiles(backup, extract, filter, files)
+          } else {
+            resolve(files)
```

Comment:
This is interesting - IIRC the extract functionality was kind of added as an afterthought during the initial write

I can definitely see both cases where you'd want either the report output or the log output during the extract depending on if it's being run manually vs. in a script

Reply:
Might be worth it to add it as an option in main task runner via some sort of command line flag to disable report printing alltogether (perhaps it could default to hide reports if we're running an extract inside of a tty, but allow it to be flipped?)

Reply:
That would be an option yeah. For now I think we should probably defer that decision and integrate the rest of the changes, since they improve on various other parts of file exports.